mappings/net/minecraft/block/FireBlock.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/class_137 net/minecraft/block/FireBlock
+	FIELD field_278 ENCOURAGEMENT [I
```

Comment:
Is that what upstream yarn calls it? Shouldn’t it be something like spreadability?

Reply:
this is what the minecraft wiki calls it, I'll change it

Reply:
done in cfd2745